src/uwtools/config_validator.py

```diff
@@ -2,61 +2,70 @@
 Support for validating a config using JSON Schema.
 """
 import json
-import os
+from pathlib import Path
+from typing import List
 
 import jsonschema
 
 from uwtools.config import YAMLConfig
 from uwtools.logger import Logger
 
+# Public functions
 
-def config_is_valid(config_file: str, validation_schema: str, log: Logger) -> bool:
+
+def config_is_valid(config_file: str, schema_file: str, log: Logger) -> bool:
     """
-    Validate a config using JSON Schema.
+    Check whether the given config file conforms to the given JSON Schema spec and whether any
+    filesystem paths it identifies do not exist.
     """
-    # Get the config file to be validated and dereference Jinja2 templates.
-    # The config file will only be 2 levels deep.
-
-    config_obj = YAMLConfig(config_file, log_name=log.name)
-    config_obj.dereference_all()
-
-    # Load the JSON Schema validation schema.
-
-    with open(validation_schema, "r", encoding="utf-8") as schema_file:
-        schema = json.load(schema_file)
-
-    # Validate the config file against the schema file.
-
-    schema_error = 0
-    validator = jsonschema.Draft7Validator(schema)
-
-    # Print out each schema error.
-
-    errors = validator.iter_errors(config_obj.data)
+    # Load the config and schema.
+    yaml_config = YAMLConfig(config_file, log_name=log.name)
```

Comment:
Ultimately, we would want to be able to validate any type of config we support (INI and namelists, too). I definitely think this name is fine matching the current capability, but we will likely want to rename it when we actually add that feature.